cmd/prysmctl/p2p/request_blobs.go

```diff
@@ -118,7 +126,28 @@ func cliActionRequestBlobs(cliCtx *cli.Context) error {
 		return errors.New("no peers found")
 	}
 	log.WithField("peers", allPeers).Info("List of peers")
-	chain, err := c.initializeMockChainService(ctx)
+
+	// Process fork flag
+	forkVersion := -1 // -1 means auto-detect based on current epoch
+	if requestBlobsFlags.Fork != "" {
+		forkVersion, err = version.FromString(strings.ToLower(requestBlobsFlags.Fork))
+		if err != nil {
+			availableForks := []string{}
+			for _, id := range version.All() {
+				availableForks = append(availableForks, version.String(id))
+			}
+			return errors.Errorf("invalid fork %q, available options: %s",
```

Comment:
```suggestion
			return errors.Errorf("invalid fork %s, available options are %s",
```
Golang errors shouldn't contain colons because they are used to separate error messages